jobserver/models.py

```diff
@@ -422,6 +423,10 @@ class Release(models.Model):
     def file_path(self, filename):
         return settings.RELEASE_STORAGE / self.upload_dir / filename
 
+    @property
+    def manifest(self):
+        return json.load(self.file_path("metadata/manifest.json").open())
```

Comment:
Same as below, won't this leave the manifest filehandler open?

Reply:
This one actually will leave a handle, which as its cpython with be closed implicitly as it's immediately dereferenced, but it is properly better to be explicit